src/zig.rs

```diff
@@ -188,7 +188,10 @@ impl Zig {
             }
         }
 
-        let mut child = Self::command()?
+        let mut zig = Self::command()?;
+        env::set_var("ZIG_COMMAND", format!("{:?}", zig));
```

Comment:
It's `"zig"` or `"python3" "-m" "ziglang"`. I think we should get rid of the double quotes.